docs/en/mutations.md

```diff
@@ -138,8 +138,70 @@ Now imagine we are debugging the app and looking at the devtool's mutation logs.
 
 ### Committing Mutations in Components
 
-You can commit mutations in components with `this.$store.commit('xxx')`, or use the `mapMutations` helper which maps component methods to `store.commit` calls (requires root `store` injection):
+Vuex mutations are not directly available as callable methods inside your Vue templates. To trigger mutations from your templates, expose them via your methods.
```

Comment:
Hmm, I prefer previous version for explaining how to trigger the mutations in components. Because it would be clear that the store and the component template are separated. Also, it is technically available since the users can call `$store.commit('xxx')` in template.